client/app/data/l10n/ca.json

```diff
@@ -596,5 +596,6 @@
   "This email is to remind you the presence of unread or updated reports.": "Aquest correu electrònic us recorda la presència d’informes no llegits o actualitzats.",
   "Reminder about unread or updated reports": "Recordatori sobre els informes no llegits o actualitzats",
   "Role: {Role}": "Rol: {Role}",
-  "Password: {Password}": "Contrasenya: {Password}"
+  "Password: {Password}": "Contrasenya: {Password}",
+  "Custom support URL": "URL de suport personalitzada"
```

Comment:
this manual change to this file should not be included. after the pull request we will use some grunt commands to automatically generate these files.

grunt makeTranslationsSource will update the en.po file
grunt updateTranslations will update the *.json files